MonMaraicher/Features/Map/MapViewModel.swift

```diff
@@ -42,6 +44,28 @@ final class MapViewModel: ObservableObject {
         CLLocationManager().requestWhenInUseAuthorization()
         CLLocationManager().desiredAccuracy = kCLLocationAccuracyBest
     }
+
+    func onButtonTapped() {
+        if let nearbyFarmer = findNearbyFarmer() {
+            mapCameraPosition = .region(MKCoordinateRegion(center: CLLocationCoordinate2D(latitude: nearbyFarmer.location.latitude, longitude: nearbyFarmer.location.longitude), span: MKCoordinateSpan(latitudeDelta: 0.01, longitudeDelta: 0.01)))
```

Comment:
It's hard to read.. you are initializing 2 things even 3.. all on the same line.

You can create constants and indent all parameters with return to line alignement to make it easier to read